Source/DafnyLanguageServer/Workspace/ProjectManagerDatabase.cs

```diff
@@ -49,10 +49,7 @@ public class ProjectManagerDatabase : IProjectDatabase {
     }
 
     public async Task UpdateDocument(DidChangeTextDocumentParams documentChange) {
-      var success = fileSystem.UpdateDocument(documentChange);
-      if (!success) {
-        throw new ArgumentException("the document change ranges did not match ranges inside the documents");
-      }
+      fileSystem.UpdateDocument(documentChange);
```

Comment:
Removing this success check seems like a deal breaker to me. Unless we can prove it is never reached? If there is any inconsistency, I'd prefer to restart VSCode rather than depending on code that is not in sync with what I see.

Reply:
It moved to here: https://github.com/dafny-lang/dafny/pull/5246/files#diff-25ce58f900ec985827ecd86d05951902a283fdb41851bf1c13e1e2e61acf3e76